packages/shared/src/components/cards/WelcomePostCardFooter.tsx

```diff
@@ -17,15 +17,31 @@ export const WelcomePostCardFooter = ({
     [post?.contentHtml],
   );
 
-  if (post.image) {
+  const image = useMemo(() => {
+    if (post?.image) return post?.image;
+
+    const parser = new DOMParser();
+    const doc = parser.parseFromString(post?.contentHtml, 'text/html');
```

Comment:
Also non blocking but just make sure if for ever reasons `post?.contentHtml` is `undefined` we want to make sure that `parseFromString` does not break.

Reply:
Have tested and it doesn't break if it's undefined